Client/Frontend/Components/ActionFooterView.swift

```diff
@@ -0,0 +1,91 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import UIKit
+
+struct ActionFooterViewModel {
+    let title: String
+    let actionTitle: String
+    let a11yTitleIdentifier: String = AccessibilityIdentifiers.ActionFooter.title
+    let a11yActionIdentifier: String = AccessibilityIdentifiers.ActionFooter.primaryAction
+}
+
+final class ActionFooterView: UIView, ThemeApplicable {
+    private struct UX {
+        static let labelSize: CGFloat = 13
+        static let buttonSize: CGFloat = 13
+        static let padding: CGFloat = 8
+        static let containerMargins = UIEdgeInsets(top: 8, left: 8, bottom: 8, right: 8)
+    }
+
+    private lazy var titleLabel: UILabel = .build { label in
+        label.font = DefaultDynamicFontHelper.preferredFont(
+            withTextStyle: .footnote,
+            size: UX.labelSize)
+        label.numberOfLines = 0
+        label.setContentCompressionResistancePriority(.required, for: .vertical)
+    }
+
+    private lazy var primaryButton: ResizableButton = .build { button in
+        button.titleLabel?.font = DefaultDynamicFontHelper.preferredFont(
+            withTextStyle: .footnote,
+            size: UX.buttonSize)
+        button.titleLabel?.numberOfLines = 1
```

Comment:
We should not limit the number of lines here - in other languages the link might be longer and limiting it makes the button less accessible.